mm-project/qt_painter
# Fix bugs (#112)

* fix Remove capstyle,joinstyle radiobuttons, remove pen&brush combo & make new radio buttons instead

* Dev (#109)

* fix Remove capstyle,joinstyle radiobuttons, remove pen&brush combo & make new radio buttons instead (#104)

* Delete shape (#101)

* Integrating from dev (gigantic merge) (#67)

* (infrastcture,load,save,qa,gui,new,close) Giant merge (#65)

* improved command infrastructure
* adds sqa comparing functionality
* new test-cases and harness scripts
* interactive load save new close
* directive load save new
* adds stack dump
* better logging replaying
* modal dialog handling

* Update rq_object.hpp (#66)

* Revert "Integrating from dev (gigantic merge) (#67)" (#68)

This reverts commit a3ada3e3d806b8ecb4798afd7af40a7ff3a0c5e8.

* fixed compile, console -> 2 tabs

* fix compile

* initial draft for delete command

* review

* compiler warnings

* fix rq line selection

* fix rq line selection

* clicking on button in ribbon tab should fix it's on/off position #70

* Add new Tools Ribbontab and add Tools ->Console action #91

* review

* delete

* delete
